src/sentry/static/sentry/app/views/organizationEmpty.jsx

```diff
@@ -0,0 +1,92 @@
+import {Link} from 'react-router';
+import React from 'react';
+import styled from 'react-emotion';
+
+import {t} from '../locale';
+import AsyncView from './asyncView';
+import ConfigStore from '../stores/configStore';
+import NarrowLayout from '../components/narrowLayout';
+import space from '../styles/space';
+import withOrganizations from '../utils/withOrganizations';
+
+class OrganizationEmpty extends AsyncView {
+  getTitle() {
+    return 'No Organizations';
+  }
+
+  renderBody() {
+    let {organizations} = this.props;
+    let features = ConfigStore.get('features');
+    let canCreateOrg = features.has('organizations:create');
+    let pendingDeleteOrganizations = organizations.filter(
+      ({status}) => status.id === 'pending_deletion'
+    );
+
+    return (
+      <NarrowLayout>
+        <h3>{t('No Organizations')}</h3>
+
+        <p>
+          {t(
+            'You are not a member of any active organizations, your team may need to invite you to an existing organization. '
+          )}
+        </p>
+
+        <LinksWrapper>
+          <LinksChild>
+            <h5>{t('Quick Links')}</h5>
+            <ul>
+              {canCreateOrg && (
+                <li>
+                  <Link to="/organizations/new/">{t('Create new organization')}</Link>
+                </li>
+              )}
+              <li>
+                <a href="/account/settings/">{t('Account Settings')}</a>
+              </li>
+              <li>
+                <a href="/auth/logout/">{t('Logout')}</a>
+              </li>
+            </ul>
+          </LinksChild>
+          {!!pendingDeleteOrganizations.length && (
+            <LinksChild>
+              <h5>{t('Organizations Pending Delete')}</h5>
+              <p>
+                {t(
+                  'Below are organizations that will be deleted. Follow the links to restore an organization.'
+                )}
+              </p>
+              <ul>
+                {pendingDeleteOrganizations.map(org => (
+                  <li key={org.slug}>
+                    <Link to={`/${org.slug}/`}>{org.slug}</Link>
+                  </li>
+                ))}
+              </ul>
+            </LinksChild>
+          )}
+        </LinksWrapper>
+      </NarrowLayout>
+    );
+  }
+}
+
+export {OrganizationEmpty};
+export default withOrganizations(OrganizationEmpty);
+
+const LinksWrapper = styled('div')`
```

Comment:
Do we need a custom styled component? Don't we have `grid-emotion` for this purpose?

Reply:
Hmm @Chrissy has expressed some displeasure with `grid-emotion` (esp with padding/margin props).

Reply:
I don't like that `grid-emotion` is just a css-truncation machine slash presentational div maker. It can also be very hard to read and un-DRY, which seems like the wrong direction since styled components are already easy to read and pretty DRY.

But I'm not stubbornly against it. My main beef was that it wasn't a complete solution to our spacing-scale problem, but `space()` helps accomplish that and we can just replace `grid-emotion` with something that uses `space()` soon.

So i guess 🤷‍♀️ ? 

@lynnagara what are your thoughts?

Reply:
Why is it un-DRY? It seems to me more DRY / less code than defining a new styled component with custom CSS in every place we want a simple div or flexbox with some spacing/padding like here. I think we can bulk replace the library with something based on our own spacing units, but I kinda like how it works in general.

Reply:
Yeah you are probably right. For one-offs it can be pretty nice. I think the spots I'm thinking of are spots where we should use a styled component and don't:

![image](https://user-images.githubusercontent.com/435981/38892124-33dad10e-423b-11e8-8cb7-b7a5fa6f79ef.png)

I think I'm cool with using `Box` when it's appropriate. Personally, I still find pure css much easier to read, and I'm a _little_ worried about media queries, especially for stuff like `w={300}` but I do like the idea of continuing to modularize `Flex` since it is so dang repetitive, and named one-off styled components can also be a drag.

Reply:
`Box` and `Flex` accept arrays which map to various breakpoints, we used it in a couple of places but not many. I think it could be a way for us to better enforce consistent breakpoints in the app

Reply:
Sure. It's only a partial solution, and we'll need the media queries for other stuff besides what `grid-emotion` supports, but I don't see why everything can't work together.

Reply:
@lynnagara Updated https://github.com/getsentry/sentry/pull/7984/commits/5b9dd806d71b728d23a8f1ce1b79f27df4aed1f7